fastify.js

```diff
@@ -259,6 +259,10 @@ function build (options) {
     return fastify
   }
 
+  function exposeRoutes () {
+    return map.entries()
+  }
```

Comment:
this should not be a high-traffic method, and it can be cached. I prefer to not have it tied fo the internal representation as much as possible to